smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/util/SimpleResultSyncPoint.java

```diff
@@ -29,4 +31,14 @@ public void signalFailure() {
     public void signalFailure(String failureMessage) {
         signal(new Exception(failureMessage));
     }
+
+    public static void assertSuccess(SimpleResultSyncPoint ResultSyncPoint, long timeout, String message) throws InterruptedException {
+        try {
+            ResultSyncPoint.waitForResult(timeout);
```

Comment:
Drop the unnecessary `ResultSyncPoint` qualification. Furthermore, return the result of  `waitForResult()` in the method. There is no need to swallow it.

```suggestion
            return waitForResult(timeout);
```

Reply:
`waitForResult()` is not a static method. It's invoked on the argument. The argument name should not start with a capital letter, which I've now fixed.

Convention for assertions implementations is that they either throw an `AssertionException` in case of failure, or do nothing (see `org.junit.jupiter.api.Assertions` for example). I've never seen an assertion that returns a value. That would allow for funky usage patterns (and possibly, confusing ones). Let's not break convention.